src/pages/Trade/components/TradePairOrder/styled.tsx

```diff
@@ -58,18 +58,25 @@ export const PairOrderFormBox = styled.div`
     }
   }
   .submit-item {
-    padding: 0;
+    padding: 0 !important;
     margin: 0;
     height: 60px;
     text-align: center;
     .submitBtn {
+      padding: 0;
       font-size: 17px;
       font-weight: 500;
       height: 60px;
       width: 100%;
       color: rgba(0, 106, 151, 1);
       text-align: center;
     }
+    .submitBtn[disabled] {
```

Comment:
Why some of the class names are in camel case and some are in underscore form?

Reply:
Checke all files className and use the camel case.